source/localizable/smart_cart/webhook.html.md.erb

```diff
@@ -326,7 +509,68 @@ Name           | Type   | Value | Description
     ],
     "created_at": "2019-10-29T10:39:23+02:00",
     "expires_at": "2019-10-29T16:39:23+02:00",
-    "dispatch_until": "2019-10-30T15:00:00+02:00"
+    "dispatch_until": "2019-10-30T15:00:00+02:00",
+    "accept_options": {
```

Comment:
We shouldn't add `accept`/`reject` options for courier voucher creation payloads. 
The `state` attribute within the order payload should also be `accepted`, not `open`. Missed this in an earlier review :)

Reply:
fixed thanks